pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -719,3 +519,23 @@ func (exp *flowExporter) findFlowType(conn flowexporter.Connection) uint8 {
 	klog.Warningf("Source IP: %s doesn't exist in PodCIDRs", conn.FlowKey.SourceAddress.String())
 	return 0
 }
+
+func (exp *flowExporter) exportConn(conn flowexporter.Connection) error {
```

Comment:
same as other comment, `conn` should probably be a pointer

Reply:
I guess we can't separately call `addConnToSet` to a list of conns, and then call `sendDataSet` for each of the set? @srikartati Is that correct?
As we need to do the export process(`addConnToSet` - `sendDataSet`) of connection items one by one, and we want to make sure not holding the lock while doing I/O(`sendDataSet`), we'll need to release the lock from the beginning of export? so we currently use copy here instead of pointer.

Reply:
Sorry, I'm confused. The calls to `GetExpiredConns` already create copies of the connections that need to be exported. Why do we need to make another copy when calling `exportConn` and `addConnToSet`?

It would be different if `GetExpiredConns` was returning a list of connection *keys* but that's not the case here.

Reply:
sorry my bad. I didn't get your point previously. I have changed the argument of `exportConn` and `addConnToSet` to pointer. Thanks!